extensions/rich_text_components/Collapsible/directives/collapsible.directive.html

```diff
@@ -11,3 +11,50 @@
     </div>
   </div>
 </uib-accordion>
+
+<!-- TODO(ankita240796): This styling is required since ui-bootstrap does not work correctly with bootstrap 4. We need to switch from ui-bootstrap to ng-bootstrap once we have upgraded the collapsible component to Angular 8.-->
```

Comment:
We are following a new pattern for todo comments:

`# TODO(#issueID): Description`

Can you please add an issue ID instead?

Reply:
Done.